LEAF_Nexus/sources/Data.php

```diff
@@ -419,6 +423,10 @@ public function modify($UID)
 
     public function getAllTags($uid)
     {
+        if(!is_numeric($uid)) {
```

Comment:
The assertion of `!is_numeric($UID)` should be removed since there is no guarantee that all classes that inherit from `Data.php` will have numeric UIDs.